public/js/employeeDetails.js

```diff
@@ -0,0 +1,84 @@
+const links = document.querySelectorAll('.tablink');
+const singleBody = document.querySelectorAll('.singleBody');
+
+const sendData = (form) => {
+  const formData = new FormData(form);
+  const holdData = {};
+  formData.forEach((value, key) => {
+    holdData[key] = value;
+  });
+  fetch('/employee', {
+    method: 'PUT',
+    body: JSON.stringify(holdData),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  }).then((result) => result.json())
+    .then((result) => {
+      if (result.err) {
+        return swal('', result.err, 'error');
+      }
+      return swal('', result.message, 'success');
+    }).catch(err => swal('', err, 'error'));
+};
+
+const show = (event, divName) => {
+  links.forEach((link) => {
+    link.className = link.className.replace('tabActive', '');
+  });
+  singleBody.forEach((body) => {
+    body.style.display = 'none';
+  });
+  document.getElementById(divName).style.display = 'block';
+  event.currentTarget.className += ' tabActive';
+};
+
+const editBtn = document.querySelectorAll('.editbtn');
+editBtn.forEach((x) => {
+  x.addEventListener('click', (event) => {
+    const divParent = event.target.parentElement;
+    const elements = divParent.querySelectorAll('.input-field');
+    const btnSave = divParent.querySelector('.btn-save');
```

Comment:
don't use `class` name for selection in Dom if you need to target a single element
use `id` instead of

Reply:
In this case we need to use class name because we have 4 forms and each form has button.
It's better to put a class in the button in each form than put id.

Reply:
:+1: